src/components/Payroll/helpers.test.ts

```diff
@@ -902,4 +904,72 @@ describe('Payroll helpers', () => {
       })
     })
   })
+
+  describe('getPayrollType', () => {
```

Comment:
Nit: it may be time to convert `helpers` into a directory and start placing utils in smaller files 😅

Reply:
True!